### PR TITLE
Add aes-js@3.0.0 w/ npm auto-update

### PR DESCRIPTION
**Pull request by zduerr**
Pull request for issue: #10903 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/ricmoo/aes-js
 * Official website (optional, not the repository): 
 * NPM package url (optional): aes-js
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 15
   - Count of stars: 200
   - Count of forks: 45
 * NPM download stats (optional):
   - Downloads in the last day: 1,219
   - Downloads in the last week: 5,887
   - Downloads in the last month: 26,646

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/aes-js/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "aes-js",
+  "version": "3.0.0",
+  "description": "A pure JavaScript implementation of the AES block cipher and all common modes of operation.",
+  "filename": "index.js",
```
Comment (hare1039) [inline ajax/libs/aes-js/package.json]: Can you also add a minified `index.min.js`?
Also change `"filename"` to `"index.min.js"`

Our recommend minify tool: https://github.com/PeterDaveHello/web-minify-helper

File: ajax/libs/aes-js/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    "decrypt",
+    "block",
+    "cipher"
+  ],
+  "author": "Richard Moore <me@ricmoo.com>",
+  "license": "MIT",
+  "npmName": "aes-js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
```
Comment (hare1039) [inline ajax/libs/aes-js/package.json]: The file structure doesn't have a `dist` folder, so please change to the following 
```
"basePath": "",
"files": [
  "index.js"
]
```

**Comment by zduerr** [comment]
Fixed the issues, sorry about that!

File: ajax/libs/aes-js/package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    "cipher"
+  ],
+  "author": "Richard Moore <me@ricmoo.com>",
+  "license": "MIT",
+  "npmName": "aes-js",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "**/*"
```
Comment (extend1994) [inline ajax/libs/aes-js/package.json]: Please use `index.js` because we only need this file. Thanks.

**Comment by extend1994** [comment]
@cdnjs/intern3 Can you please help the @zduerr rebase the branch? Thanks. (My Internet speed is too slow to do this)

**Comment by PeterDaveHello** [comment]
@extend1994 let me know how slow is your Internet access in private, I'll see how can I help, thanks.

**Comment by PeterDaveHello** [comment]
@cdnjs/intern3 no updates for 3 days?

**Comment by hare1039** [comment]
@extend1994 @clairetsai818 Are your part OK?

**Pull request merged**